packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoPageHeader/WfoPageHeader.tsx

```diff
@@ -51,10 +51,10 @@ export const WfoPageHeader: FC<WfoPageHeaderProps> = ({
                 </EuiHeaderSectionItem>
 
                 <EuiButtonIcon
-                    aria-label="Logout"
+                    aria-label="Show/Hide side menu"
```

Comment:
Do we want translations for aria labels as well?

Reply:
Don't think this is necessary.